README.md

```diff
@@ -414,5 +414,10 @@ In order to create an image using the Red Hat Data Grid server, it's necessary t
 cekit build --overrides-file dg-override.yaml docker
 ```
 
+### Debug Container Data Grid
```

Comment:
@diegolovison Thanks for the PR. Can you add a sentence explaining what this allows the user todo.